src/web/wms/src/main/resources/GeoServerApplication.properties

```diff
@@ -161,7 +161,8 @@ WMSAdminPage.cacheEntrySize = Max entry size (Kb)
 WMSAdminPage.remoteStyleTime = Remote Styles Time Settings
 WMSAdminPage.remoteStyleTimeout = Timeout in milliseconds
 WMSAdminPage.remoteStyleMaxRequestTime = Max request time in milliseconds
-
+WMSAdminPage.includeDefaultGroupStyleInCapabilities = Include Default LayerGroup Style In GetCapabilities
+WMSAdminPage.getCapabilitiesSettings = GetCapabilities Settings
```

Comment:
The WMS page has plenty of capabilities specific settings, if we go for this title, then they should all be grouped toghether. Examples follow.

Root layer and limited SRS list:

![image](https://user-images.githubusercontent.com/325433/118466447-2779a300-b703-11eb-96e3-b32b1c55223a.png)

Scale hint:

![image](https://user-images.githubusercontent.com/325433/118466499-33fdfb80-b703-11eb-8b31-c53b9c608bd0.png)

And well, also the "Service metadata" at the top of the page, it also shows in caps.
Maybe go for "Layer groups capabilities settings" instead?

Reply:
fixed

Reply:
fixed